projects/transactions-journey/src/lib/pipes/filter-transactions.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Transaction } from '../model/transaction';
+
+@Pipe({
+  name: 'filterTransactions'
+})
+export class FilterTransactionsPipe implements PipeTransform {
+
+  transform(value: Transaction[], text: string): Transaction[] {
```

Comment:
Should there be a unit test for this pipe?

Reply:
I will add unit tests as part of the scope of this ticket if you don't mind https://backbase.atlassian.net/browse/WF-336